### DISCUSSION TITLE
Allow Rivet documentation to be downloaded as Markdown/text files for ChatGTP

### DISCUSSION THREAD

**Discussion by heyheni** [Ironclad/rivet#107] [Ideas]
Hi! 
Rivet's documentation is incredibly helpful and well-written. However, it would be extremely useful if the documentation could be downloaded as Markdown files or plain text so that ChatGTP can read them and answer questions. **Is there already a link like rivet.ironcladapp.com/docs/documentationforLLM.txt that you could paste into ChatGTP?**

Thank you! Looking forward to create my dream bicycle tour planer chatbot without much coding knowledge. 

PS: you could build a chat bot directly into the documentation to showcase your tools capabilities?

**Comment by abrenneke** [comment]
Sorry for the delay, but that's a great idea! I think the docs in total would be too big for ChatGPT, I've experimented with it a bit. Would need to use RAG to get acceptable Q&A I think.

**Comment by tonyoconnell** [comment]
You can use GPT Crawler from Builder.io https://github.com/BuilderIO/gpt-crawler
